util/combine_csv.py

```diff
@@ -0,0 +1,72 @@
+from calendar import c
```

Comment:
Please add a header with:
- copyright
- how to use this utility.

Reply:
This was for the Karnataka Demo, with some more functionalities like renaming columns in each file. I can drop it if not required.